xarray/coding/calendar_ops.py

```diff
@@ -0,0 +1,283 @@
+from datetime import timedelta
+
+import numpy as np
+
+from ..core.common import is_np_datetime_like
+from .cftime_offsets import date_range_like, get_date_type
+from .times import (
+    _is_numpy_compatible_time_range,
+    _is_standard_calendar,
+    cftime_to_nptime,
+    convert_cftimes,
+)
+
+try:
+    import cftime
+except ImportError:
+    cftime = None
+
+
+def _days_in_year(year, calendar, use_cftime=True):
+    """Return the number of days in the input year according to the input calendar."""
+    return (
+        (
+            get_date_type(calendar, use_cftime=use_cftime)(year + 1, 1, 1)
+            - timedelta(days=1)
+        )
+        .timetuple()
+        .tm_yday
+    )
+
+
+def convert_calendar(
+    ds,
+    target,
```

Comment:
Should we call this `calendar` to align with the description in the docstring?
```suggestion
    calendar,
```